bin/find-llvm-config

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+if ! LLVM_CONFIG=$(command -v "$LLVM_CONFIG"); then
+  llvm_config_version=$(llvm-config --version 2>/dev/null)
+  for version in 8.0 7.1 6.0 5.0 4.0 3.9 3.8; do
+    LLVM_CONFIG=$(
+    command -v llvm-config-${version%.*} || \
+    command -v llvm-config-$version || \
+    command -v llvm-config${version%.*}${version#*.} || \
+    command -v llvm${version%.*}-config || \
+    [ "${llvm_config_version#$version}" = "$llvm_config_version" ] || command -v llvm-config \
```

Comment:
It's like `lchop`: if the string isn't changed, which means the version isn't included in `llvm_config_version`, stop.
`case` isn't used because can't be parsed by Bash 3 when inside `$()`, used in macOS.